crates/matrix-sdk-indexeddb/Cargo.toml

```diff
@@ -36,6 +37,7 @@ thiserror = "1.0.30"
 tracing = "0.1.34"
 wasm-bindgen = { version = "0.2.80", features = ["serde-serialize"] }
 web-sys = { version = "0.3.57", features = ["IdbKeyRange"] }
+js-sys = { version = "0.3.58" }
```

Comment:
Please order alphabetically (I'll look into making this a CI check).

Reply:
honestly not in favor of more ci-checks on minor style things (e.g. alphabetical order of dependencies in `Crates.toml`-files ... they are not that long...) that take more CI runs to fix and don't have an automated fixing mechanism that runs in seconds....